lms/config.py

```diff
@@ -1,8 +1,8 @@
 """Configuration for the Pyramid application."""
 
 import os
+from collections.abc import Callable
```

Comment:
typing.Callable
Deprecated alias to collections.abc.Callable.

https://docs.python.org/3/library/typing.html#typing.Callable